marinerescue-frontend/src/pages/HomePage.js

```diff
@@ -1,14 +1,137 @@
+                            </div>
+                            <div class="column">
+                                <div class="content">
+                                    <img id="icon" src="images/humphrey.png"></img>
+                                    <h3>Humphrey</h3><br />
+                                </div>                            
+                            </div>
+                            <div class="column">
+                                <div class="content">
+                                    <img id="icon" src="images/sqwacky.png"></img>
+                                    <h3>Sqwacky</h3><br />
+                                </div>                            
+                            </div>
+                            <div class="column">
+                                <div class="content">
+                                    <img id="icon" src="images/jade.png"></img>
+                                    <h3>Jade</h3><br />
+                                </div>                            
+                            </div>
+                        </div>
+                    </div>
+                </div>
+
+                <div class="section">
+                    <div class="section-4">
+                        <img src="images/two-jade-parents-background.png"></img>
+                        <button id="parents-button"><h1>Parents of Marine Rescuers</h1>Read more about our team and mission in educating young scientists through Marine Rescue, a citizen science project teaching debris’ impacts on marine wildlife. </button>
+                    </div>
+                </div>
+
+                <div class="section">
+                    <div class="footer">
+                    <div class="row">
+                            <div class="column">                        
+                            </div>
+                            <div class="column">
+                                <h3>Connect with us: </h3>
+                                <img id="icon" src="images/youtube-logo.png"></img> <br />
+                            </div>
+                            <div class="column">
+                            </div>
+                        </div>
+                        <div class="row">
+                            <div class="column">
+                                <div class="navbar-logo">
+                                    <img src="images/logo.png"></img>
+                                </div>                            
+                            </div>
+                            <div class="column">
+                                <h3>@2021 AppleJACK</h3>
+                            </div>
+                            <div class="column">
+                                <img id="icon" src="images/ischool-logo.png"></img> <br />
+                                <img id="icon" src="images/coasst-logo.png"></img>
```

Comment:
class, alt, self-closing
```suggestion
                                <img class="icon" src="images/ischool-logo.png" alt="Logo of University of Washington iSchool" /> <br />
                                <img class="icon" src="images/coasst-logo.png" alt="Logo of University of Washington iSchool" />
```